recipe/meta.yaml

```diff
@@ -19,11 +23,15 @@ requirements:
     - setuptools
   run:
     - python
+    - certifi
 
 test:
   imports:
     - libcloud
 
+  commands:
+    - python -c "import sys; from libcloud.security import CA_CERTS_PATH; import certifi; ok = 0 if certifi.where() in CA_CERTS_PATH else 1; sys.exit(ok)"
```

Comment:
Agree, just wanted to make sure I understood what you wanted. It should be done now.